crates/libcgroups/src/v1/cpu.rs

```diff
@@ -40,6 +41,7 @@ impl Controller for Cpu {
                 || cpu.quota().is_some()
                 || cpu.realtime_period().is_some()
                 || cpu.realtime_runtime().is_some()
+                || cpu.idle().is_some()
```

Comment:
Should either add or adapt the test cases to account for setting and checking this new setting.